src/Smidge.Core/IRequestHelper.cs

```diff
@@ -27,8 +26,8 @@ public interface IRequestHelper
         /// Returns the compression type for the current request
         /// </summary>
         /// <returns></returns>
-        CompressionType GetClientCompression(IDictionary<string, StringValues> headers);
```

Comment:
This will be a binary breaking change. Is there any way to do this without breaking the interface?

Reply:
```GetCommaSeparatedValues``` is only implemented for ```IHeaderDictionary```. I can reimplement that method if you'd like, but the breakage is should be very limited as ```IHeaderDictionary``` is also an ```IDictionary<string, StringValues>```. Only people reimplementing ```IRequestHelper``` will need to make a small change. Headers from .NET already implement `IHeaderDictionary`, which is why there are so few changes.

Reply:
I know it doesn't seem like much of a change but a binary breaking change will require a new major version release else people that upgrade will get ysods if they are using another implementation on the older version. So we need to have non-binary breaking changes.

Reply:
> will get ysods
Only if they implement `IRequestHelper`, which I suspect will not be a lot 😊 But I have changed the implementation.

Reply:
Thanks. Prob not but in my experience in the Umbraco OSS project, any undocumented binary changes can cause huge headaches upstream for some live projects and its better to not assume :)